### PR TITLE
TIP-890: Fix Standard for 3.0

### PR DESCRIPTION
**Pull request by jjanvier**
2. try to install the standard version on your computer (I couldn't do it due to `composer` problems)
3. check the PIM works properly once installed

### CODE REVIEW

File: app/PimRequirements.php [hunk 1]
```diff
@@ -64,7 +64,6 @@ public function getPimRequirements(): array
     }
 
     /**
-<<<<<<< HEAD
```
Comment (arnolanglade) [inline app/PimRequirements.php]: Problem with github ?

File: app/AppKernel.php [hunk 2]
```diff
@@ -72,30 +72,26 @@ protected function getPimBundles()
...
-            new Pim\Bundle\FilterBundle\PimFilterBundle(),
+            new Oro\Bundle\PimFilterBundle\PimFilterBundle(),
+            new Oro\Bundle\PimDataGridBundle\PimDataGridBundle(),
             new Akeneo\UserManagement\Bundle\PimUserBundle(),
 
             // Channel bundles
             new Akeneo\Channel\Bundle\AkeneoChannelBundle(),
 
             // PIM bundles
+            new Pim\Bundle\EnrichBundle\PimEnrichBundle(),
```
Comment (arnolanglade) [inline app/AppKernel.php]: it should be removed later, right?

File: app/config/config.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 imports:
-    - { resource: '@PimEnrichBundle/Resources/config/pim.yml' }
+    - { resource: '../../vendor/akeneo/pim-community-dev/src/Akeneo/Platform/config/pim.yml' }
```
Comment (arnolanglade) [inline app/config/config.yml]: :+1:

File: composer.json [hunk 4]
```diff
@@ -11,13 +11,10 @@
     ],
     "autoload": {
         "psr-0": {
-            "": "src/",
-            "Context": "features/",
-            "Pim\\Behat": "vendor/akeneo/pim-community-dev/features/"
+            "": "src/"
```
Comment (arnolanglade) [inline composer.json]: :+1:

File: app/PimRequirements.php (hunk 1 above)
Comment (jjanvier) [reply app/PimRequirements.php]: "<<<<<<< HEAD" was in the file itself
I just remove it
I probably comes from a bad merge

**Comment by anaelChardan** [comment]
Look like it works like a charm

**Pull request merged**